Automattic/wp-calypso
# Comments: add option to poll for new comments in the background

This PR adds the plumbing for loading new comments in the background. We poll every 60 seconds, and new comments are saved in Redux state. It's initially enabled on Reader full post in development mode only.

A later PR will expose a UI for loading pending comments.

### To test

Load a Reader full post, like:

http://calypso.localhost:3000/read/blogs/82798297/posts/82

Make a new comment (or several comments) on the original post in a separate tab:

https://futonbleu.wordpress.com/2016/04/18/comment-test/

Watch the network tab on the original page. You should see a request go to the replies endpoint every 60 seconds. The first response after you submitted your new comment(s) should contain the new comments in the response (and should not contain any earlier comments).

<img width="997" alt="screen shot 2018-08-02 at 18 04 31" src="https://user-images.githubusercontent.com/17325/43565377-9aafff52-967e-11e8-97c1-a685cefaa07a.png">

In your console, type `state.comments.pendingItems`. Ensure that the returned object contains your new comments only.

<img width="1188" alt="screen shot 2018-08-02 at 18 04 59" src="https://user-images.githubusercontent.com/17325/43565384-9ec89f9a-967e-11e8-8d6b-2a324c765e8b.png">
